packages/regl-worldview/src/index.js

```diff
@@ -6,7 +6,7 @@
 //  found in the LICENSE file in the root directory of this source tree.
 //  You may not use this file except in compliance with the License.
 
-import Worldview from "./Worldview";
+import Worldview, { WorldviewBase as OffscreenWorldview } from "./Worldview";
```

Comment:
The main difference here is that `WorldviewBase` doesn't use a `ContainerDimensions`? I can't think of a better name, but I feel like that's the important difference -- not necessarily how we use it. But this name is ok too.

Reply:
That's correct. We need `WorldviewBase` because it's not using `ContainerDimensions`. I can't find a better name either. The alternative was to just export it as `WorldviewBase`

Reply:
Maybe `ControlledSizeWorldview` or `FixedSizeWorldview`?